extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -65,9 +84,25 @@ export class AmpSelector extends AMP.BaseElement {
       this.element.setAttribute('aria-disabled', 'true');
     }
 
+    const kbSupportMode = this.element.getAttribute('keyboard-support');
```

Comment:
Every code we write runs along side existing code, we can't add experimental flags for everything we do. This is a small and isolated feature that does not impact the existing functionality of the component, is fully opt-in, API is not expected to change and there is no extra testing planned. I really don't think it needs an experiment flag.